doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -384,297 +381,248 @@ Rewriting with definitional equality
-   + While reducing a constant unfolding to (co)fixpoints, the tactics
-     use the name of the constant the (co)fixpoint comes from instead of
+   + When reducing a constant unfolding to (co)fixpoints, the tactic
+     uses the name of the constant the (co)fixpoint comes from instead of
      the (co)fixpoint definition in recursive calls.
 
-   The :tacn:`cbn` tactic was intended to be a more principled, faster and more
-   predictable replacement for :tacn:`simpl`.
-
-   The :tacn:`cbn` tactic accepts the same flags as :tacn:`cbv` and
-   :tacn:`lazy`. The behavior of both :tacn:`simpl` and :tacn:`cbn`
-   can be tuned using the :cmd:`Arguments` command.
-
-   .. todo add "See <subsection about controlling the behavior of reduction strategies>"
-      to TBA section
+   :n:`@occurrences`
+     Permits selecting whether to reduce the conclusion and/or one or more
+     hypotheses.  The `at` option of :n:`@occurrences` is not allowed here,
+     but :n:`reference_occs` and :n:`@pattern_occs` have a somewhat less
+     flexible `at` option for selecting specific occurrences.
 
-   Notice that only transparent constants whose name can be reused in the
-   recursive calls are possibly unfolded by :tacn:`simpl`. For instance a
-   constant defined by :g:`plus' := plus` is possibly unfolded and reused in
-   the recursive calls, but a constant such as :g:`succ := plus (S O)` is
-   never unfolded. This is the main difference between :tacn:`simpl` and :tacn:`cbn`.
-   The tactic :tacn:`cbn` reduces whenever it will be able to reuse it or not:
-   :g:`succ t` is reduced to :g:`S t`.
+   :tacn:`simpl` can unfold transparent constants whose name can be reused in
+   recursive calls as well as those designated by :cmd:`Arguments` :n:`@reference … /`
+   commands. For instance, a constant :g:`plus' := plus` may be unfolded and
+   reused in recursive calls, but a constant such as :g:`succ := plus (S O)` is
+   never unfolded.
 
-.. tacv:: cbn [ {+ @qualid} ]
-          cbn - [ {+ @qualid} ]
+   :n:`{| @reference_occs | @pattern_occs }` can limit the application of :tacn:`simpl`
+   to:
 
-   These are respectively synonyms of :n:`cbn beta delta [ {+ @qualid} ] iota zeta`
-   and :n:`cbn beta delta - [ {+ @qualid} ] iota zeta` (see :tacn:`cbn`).
+   - the applicative subterms whose head occurrence is the
+     constant :n:`@qualid` or is the constant used
+     in the notation :n:`@string` (see :n:`@reference`)
+   - subterms matching a pattern :n:`@one_term`
 
-.. tacv:: simpl @pattern
+   The behavior can be tuned using the :cmd:`Arguments` command.
 
-   This applies :tacn:`simpl` only to the subterms matching
-   :n:`@pattern` in the current goal.
+.. tacn:: cbn {? @reductions } @simple_occurrences
 
-.. tacv:: simpl @pattern at {+ @natural}
+   :tacn:`cbn` was intended to be a more principled, faster and more
+   predictable replacement for :tacn:`simpl`.
+   The behavior can be tuned using the :cmd:`Arguments` command.
```

Comment:
If the goal in this description is not to repeat the differences, then maybe this sentence could be removed (not sure).

Reply:
I think you meant to say "only mention the differences".  I dropped the sentence.